### DISCUSSION TITLE
Intercepting traffic on WPA2-PSK BSS with known PMK.

### DISCUSSION THREAD

**Discussion by Frostie314159** [aircrack-ng/aircrack-ng#2619] [Other wireless tools]
Hi,
While reading the security section of 802.11, I had the idea, that if you're in a network, for which you know the passphrase(like a public wifi network), you could derive all PTKs and the GTK, just by capturing the 4 way handshake.
I'm aware, that WPA2 doesn't have forward secrecy, but this renders these networks no more secure than open ones. This also makes rotating the GTK after a disassoc useless.
I figured, that if I'd find educated opinions about this, it would be here.
Let me know what you think.

**Comment by Mister-X-** [comment]
That's how airdecap-ng, and Wireshark (the decryption) works.

**Reply by Frostie314159** [reply]
Ok, didn't know this yet. Does airdecap also do this actively, by deauthing clients to force another 4 way handshake?
